pom.xml

```diff
@@ -139,6 +153,21 @@
             <version>${grpc.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+            <version>4.1.71.Final</version>
```

Comment:
netty 4.1.72 fixed the log4j security issue, why not upgrade to the newer version?